src/test/scala/com/cognite/spark/datasource/AssetsRelationTest.scala

```diff
@@ -131,20 +151,6 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
   it should "support some more partial updates" taggedAs WriteTest in {
     val source = "spark-assets-test"
 
-    val sourceDf = spark.read
```

Comment:
This probably shouldn't be removed...?

Reply:
Didn't remove it, just moved it to global scope so I could use the temp views in my test aswell.